docker-compose.yml

```diff
@@ -20,6 +20,10 @@ services:
       - 8080:80
     volumes:
       - "~/dm/docker/pgadmin:/var/lib/pgadmin"
+    environment:
+      PGADMIN_DEFAULT_EMAIL: a@b.cm
+      PGADMIN_DEFAULT_PASSWORD: 1234
+
```

Comment:
```suggestion
```

Reply:
My apologies Tobias, I didn't see this comment before I completed the pull request. I have reverted the pull request and made the change you requested. I have made another pull request to be reviewed.